applications/compressible_navier_stokes/CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
-PROJECT(compressible_navier_stokes)
+DIRNAME(PROJECT_NAME ${CMAKE_CURRENT_SOURCE_DIR})
 
-ADD_SUBDIRECTORY(couette)
-ADD_SUBDIRECTORY(euler_vortex)
-ADD_SUBDIRECTORY(manufactured_solution)
-ADD_SUBDIRECTORY(poiseuille)
-ADD_SUBDIRECTORY(shear_flow)
-ADD_SUBDIRECTORY(taylor_green)
-ADD_SUBDIRECTORY(template)
-ADD_SUBDIRECTORY(turbulent_channel)
\ No newline at end of file
+PROJECT(${PROJECT_NAME})
+
+SUBDIRLIST(SUBDIRS ${CMAKE_CURRENT_SOURCE_DIR})
+
+FOREACH(subdir ${SUBDIRS})
+  ADD_SUBDIRECTORY(${subdir})
+ENDFOREACH()
```

Comment:
Github complains here and in a number of other files that you do not have a newline at the end of the file. Would you mind making these consistent. (Single new-line at the end of file?)

Reply:
done in https://github.com/exadg/exadg/pull/113/commits/1a8206fe474f2113021abc61506534a9a75003ca